ios/FluentUI.Demo/FluentUI.Demo/Demos/SearchBarDemoController.swift

```diff
@@ -28,16 +28,59 @@ class SearchBarDemoController: DemoController, SearchBarDelegate {
         return buildBadgeView(text: "Kat Larsson", customView: imageView)
     }()
 
+    private var searchBars: [SearchBar] = []
+
+    let segmentedControl: SegmentedControl = {
+        let segmentedControl = SegmentedControl(items: [SegmentItem(title: "System"), SegmentItem(title: "Brand")],
+                                                style: .primaryPill)
+
+        // Override the selected tab's background color otherwise it will blend into the container's background
+        segmentedControl.tokenSet.replaceAllOverrides(with: [.selectedTabColor: .dynamicColor({
+            segmentedControl.fluentTheme.aliasTokens.colors[.brandForeground1Selected]
+        })])
+        return segmentedControl
+    }()
```

Comment:
Since `segmentedControl.style` is internal, could you but the segmented control on a pill button bar background?